lib/main.js

```diff
@@ -113,9 +113,30 @@ exports.mjpage = function(htmlstring, configOptions, typesetOptions, callback) {
         window.ascii.config.doc = document;
         window.ascii.PreProcess();
     }
+    //normalize type strings
+    const scripts = document.querySelectorAll('script[type^="math/"]');
+    for (let i = 0; i < scripts.length; i++) {
+        const script = scripts[i];
+        let format = script.getAttribute("type").toLowerCase().slice(5);
+        if (format === "inline-tex") {
+            format = "math/inline-TeX";
+            // via https://github.com/mathjax/MathJax/blob/master/unpacked/jax/input/TeX/jax.js#L2168
+        } else if (format.match(/tex((;|\s*|\n*)mode\s*=\s*display)?/i)) {
```

Comment:
I'm no regex expert, but I believe this will be true for `math/tex` as well. I would suggest to simply compare against `math/tex; mode=display`.